lib/plug/ssl.ex

```diff
@@ -242,9 +242,18 @@ defmodule Plug.SSL do
   end
 
   defp set_secure_defaults(options) do
-    options
-    |> Keyword.put_new(:secure_renegotiate, true)
-    |> Keyword.put_new(:reuse_sessions, true)
+    if options[:versions] == [:"tlsv1.3"] do
```

Comment:
I think that it should check whether the options contain earlier versions instead, as I assume that if there will ever be TLSv1.4 it will not support these options as well.

Reply:
I agree, this should probably be something like: 

```suggestion
    if Enum.any?([:tlsv1, :"tlsv1.1", :"tlsv1.2"], &(&1 in options[:versions])) do
```

No point in doing a MapSet dance here, is there? This is typically checked just once on server startup...

Reply:
I was wondering the same thing. But if it's not set, then we don't know whether these options should be set or not. For that I think we'd have to pick up the default with `:ssl.versions()[:supported]`...

```suggestion
  versions = Keyword.get(options, :versions, :ssl.versions()[:supported])
  
  if not Enum.any?(...) do
```

Another thing: technically `:ssl` options are not a proplist, so not sure we're supposed to use `Keyword` on it?